src/sidebar/components/navigation/navigation.tpl.js

```diff
@@ -7,13 +7,24 @@ const navigation = () => {
 
 	let back = document.createElement('button');
 	back.innerText = '⮜ Back';
-
 	back.addEventListener('click', () => {
 		history.back(); 
 	});
-
 	wrapper.appendChild(back);
-	
+
+	if (history.length <= 1 ) {
+		back.setAttribute('disabled', 'disabled')
+	}
+
+	if (window != window.top) {
+		let close = document.createElement('button')
+		close.innerText = '🗙';
+		close.addEventListener('click', () => {
+			window.parent.postMessage({type: "toggle-sidebar"}, '*')
+		});
+		wrapper.appendChild(close);
```

Comment:
hmm, doesn't this duplicate the default close button?

Reply:
this **is** the close button